Model/PayolutionBasePaymentMethod.php

```diff
@@ -282,6 +282,8 @@ public function getMetaDataFieldNames()
             'payolution_terms_url',
         ];
 
+        $aReturn = array_merge(parent::getMetaDataFieldNames(), $aReturn);
```

Comment:
Please rename `$aReturn` to `aMetaDataFields` to be more descriptive.

Reply:
👍